lib/curly/presenter.rb

```diff
@@ -307,5 +307,10 @@ def presents(*args)
     def method_missing(method, *args, &block)
       @_context.public_send(method, *args, &block)
     end
+
+    # Tells ruby (and developers) what methods we can accept.
+    def respond_to_missing?(method, include_private = false)
+      @_context.respond_to?(method, false)
```

Comment:
No; my reasoning being, we use `#public_send` above, which will error on private methods.